stat/util.lua

```diff
@@ -57,4 +57,26 @@ function util.gethostname()
   return ffi.string(name_buf)
 end
 
+function util.get_replica_host(peer)
+    local _, host = string.match(peer, '(.+)@(.+)')
```

Comment:
It seems you are strip `user[:pass]@` here. Please, comment: it is not so obvious, I think.